koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -458,6 +466,13 @@ def convert_to_parquet(self):  # noqa: C901
             return parquet_base_filename, daily_data_frames
 
         for csv_filename in file_list:
+
+            # set start date based on data in the file being processed:
+            if self.provider_type == Provider.PROVIDER_OCI:
+                self.start_date = str(csv_filename).split(".")[1]
```

Comment:
You could use `stem` or `suffixes` instead of `str`.

Reply:
I do not know the structure of this filename. I just copied this from below.